src/tools/tidy/src/style.rs

```diff
@@ -186,6 +186,10 @@ fn should_ignore(line: &str) -> bool {
     // - `//@[rev] normalize-stderr-test`
         || static_regex!("\\s*//@(\\[.*\\]) (compile-flags|normalize-stderr-test|error-pattern).*")
             .is_match(line)
+        // Matching for rustdoc tests commands.
+        || static_regex!(
+            "\\s*//@ \\!?(count|files|has|has-dir|hasraw|matches|matchesraw|snapshot)\\s.*"
```

Comment:
`ANNOTATIONS_TO_IGNORE` already contains some directives specific to htmldocck (and jsondocck I suppose). You should be able to remove them now.

Reply:
Indeed! Removed. :)

Reply:
So it was in fact needed for the line length as the CI failure indicates. XD